### ISSUE TITLE
Update reporter to show the wait count of an alias

### ISSUE THREAD

**Issue by chrisbreiding** [cypress-io/cypress#2868]
When you wait on an alias of something like a network request. (e.g. `cy.route(...).as('@getPosts')`, subsequent calls to `cy.wait('@getPosts')` will wait on subsequent responses for that request.

```javascript
cy.route(...).as('@getPosts')

cy.wait('@getPosts') // wait for 1st response
cy.wait('@getPosts') // wait for 2nd response
cy.wait('@getPosts') // wait for 3rd response
```

We should indicate this in the reporter UI in some way to make it clearer what is going on.

**Comment by jennifer-shehane** [comment]
This is duplicate of https://github.com/cypress-io/cypress/issues/477 - will close the older issue since you've already referenced in PR.

**Comment by jennifer-shehane** [comment]
Duplicate of https://github.com/cypress-io/cypress/issues/477
